e2e/tests/init_test.go

```diff
@@ -85,6 +88,17 @@ func TestMain(m *testing.M) {
 	m.Run()
 }
 
+// ethSubscription is a struct to unmarshal the JSON response from the ETH subscription.
+type ethSubscription struct {
```

Comment:
@commoddity Do you potentially have a suggestion of something we could (should?) import rather than defining it ourselves?

Reply:
@commoddity Bump